package.json

```diff
@@ -36,24 +36,28 @@
   },
   "homepage": "https://github.com/mypurecloud/webrtc-stats-gatherer#readme",
   "devDependencies": {
-    "@types/jest": "^26.0.14",
-    "jest": "^26.4.2",
-    "pre-commit": "^1.1.3",
-    "pre-push": "^0.1.1",
-    "rimraf": "^3.0.2",
-    "run-sequence": "^2.0.0",
-    "ts-jest": "^26.4.1",
-    "tslint": "^6.1.3",
-    "tslint-config-semistandard": "^8.0.1",
-    "typescript": "^4.0.3",
-    "webpack": "^4.41.2",
-    "webpack-cli": "^3.1.0"
+    "@eslint/js": "^9.2.0",
+    "@types/jest": "^29.5.12",
+    "eslint": "^8.57.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "npm-check": "^6.0.1",
+    "pre-commit": "^1.2.2",
+    "pre-push": "^0.1.4",
+    "prettier": "3.2.5",
+    "rimraf": "^5.0.7",
+    "ts-jest": "^29.1.2",
+    "typescript": "^5.4.5",
+    "typescript-eslint": "^7.9.0"
   },
   "pre-push": [
     "test"
   ],
   "pre-commit": [
-    "lint"
+    "lint",
+    "format"
   ],
-  "dependencies": {}
+  "volta": {
+    "node": "20.13.0"
```

Comment:
I added this so we had a reference for what version of node we're using. But that's also in the Jenkinsfile, so maybe we'd rather only have the one location. We could also add `engines`.

Reply:
I'm fine with leaving this but I'm not sure its really needed. Most people use NVM vs Volta so this does nothing for them.